buildenv/jenkins/JenkinsfileBase

```diff
@@ -140,6 +140,7 @@ def setup() {
 			dir( WORKSPACE) {
 				sshagent(credentials:["${params.USER_CREDENTIALS_ID}"], ignoreMissing: true) {
 					if (params.CUSTOMIZED_SDK_URL_CREDENTIAL_ID) {
+						# USERNAME and PASSWORD reference with a withCredentials block will not be visible within job output
```

Comment:
I think Jenkins file comments should be `//`or `/* */`. I am not sure `#` works.

Reply:
yep, thats what c&p gets me.

Reply:
yes, I know the build passed. But from grovvy docs http://groovy-lang.org/syntax.html, it suggests to use `//` or `/* */`. Could you change it? Thanks.